mako_config/project.config

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# project_name is always evaluated as lowercase and must be unique
+project_name: "iree"
+
+# project_alias is the display name of the project and will be shown on
+# mako.dev. It does not have to be unique.
+project_alias: "IREE"
+
+# owner_list contains a list of owners (e.g. user@google.com, mdb_group@prod.google.com, etc.).
+# Set "*" as an owner to enable global permissions.
+# owner_list: "my-mdb-group@prod.google.com"
```

Comment:
"owner_list contains a list of owners" is a useless comment, most of this references google-internal stuff, and we definitely should not enable global permissions.

```suggestion
```

But it would be useful to add a comment about what these owners are allowed to do. I assume they can create new benchmarks and owners of an individual benchmark can upload new runs to it?

Reply:
I think it's better to remove it. A project owner actually have all the permissions on project and benchmarks under the project, e.g, create/update/delete benchmarks and runs. So I think it's okay to not add a comment.